data/migrations/v1.5.0-planet_osm_point.sql

```diff
@@ -1,3 +1,12 @@
+-- Add zoom for POIs without a name
+UPDATE planet_osm_point
+  SET mz_poi_min_zoom = mz_calculate_min_zoom_pois(planet_osm_point)
+  WHERE
+    shop IS NOT NULL
+    AND name IS NULL
+    AND mz_poi_min_zoom IS NULL;
```

Comment:
Is it possible that this could alter the existing min zoom of a poi? If so, can update the condition to something like `coalesce(mz_poi_min_zoom, 999) <> coalesce(mz_calculate_min_zoom_pois(planet_osm_point), 999)`. If not, disregard :)

Reply:
Isn't the condition you gave just `mz_calculate_min_zoom_pois(planet_osm_point) <> mz_poi_min_zoom` This would be a more restrictive condition, and help cut down on the number of updates.

Reply:
It's because NULLs are "infectious" as I like to call it, ie if any of them are null then it won't update.
`select NULL <> 42` => `NULL`

Reply:
Oh yes, because it's an inequality, it should be `IS DISTINCT FROM`

Reply:
Didn't know about `is distinct from`, nice!